keras_cv/layers/augmenter.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Augmenter(object):
```

Comment:
Previously this subclasses `keras.layers.Layer`, which in theory is nice for serialization but in practice this thing shouldn't ever really be serialized, so I think this is fine.

Reply:
@ariG23498 since this is a class and not a function I think `Augmenter` is more appropriate.